stdlib/http/src/main/ballerina/src/http/resiliency/http_circuit_breaker.bal

```diff
@@ -139,7 +139,12 @@ public type CircuitBreakerClient client object {
     # + httpClient - The underlying `HttpActions` instance which will be making the actual network calls
     # + circuitHealth - The circuit health monitor
     public function __init(string url, ClientConfiguration config, CircuitBreakerInferredConfig
-                                        circuitBreakerInferredConfig, HttpClient httpClient, CircuitHealth circuitHealth) {
+    circuitBreakerInferredConfig, HttpClient httpClient, CircuitHealth circuitHealth) {
```

Comment:
Can you please check whether this is the correct formatting?

Reply:
Fixed